packages/x-components/src/x-modules/extra-params/components/snippet-config-extra-params.vue

```diff
@@ -65,7 +73,7 @@
       currency,
       ...snippetExtraParams
     }: SnippetConfig): void {
-      forEach(snippetExtraParams, (name, value) => {
+      forEach({ ...snippetExtraParams, ...this.values }, (name, value) => {
```

Comment:
Where is the test for this new feature? 🤣

Reply:
the tests are the parents